docs/reference/e2e-test.md

```diff
@@ -4,36 +4,33 @@ Run E2E test as part of github actions workflow. This E2E test will be triggered
 
 ## Architecture
 
-![architecture](../diagrams/e2e-architecture.svg)
+![architecture](../diagrams/e2e-architecture.drawio.svg)
 
 ## Steps
 
-* Pull merlin repository
-* Pull mlp repository
-* Setup go 1.17
-* Setup python 3.7
-* Setup cluster
-* Setup mlp namespace
-* Deploy mlp
-* Deploy merlin
-* Run E2E test
-
+- Pull merlin repository
+- Pull mlp repository
+- Setup go 1.17
```

Comment:
Oops missed this earlier, we have Go 1.20 now, right? :) https://github.com/caraml-dev/merlin/blob/main/api/go.mod#L3